e2e/hotkeys.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { test, expect } from '@playwright/test';
+
+import translations from '../client/i18n/locales/english/translations.json';
+
+const course =
+  '/learn/javascript-algorithms-and-data-structures/basic-javascript/comment-your-javascript-code';
+const editorPaneLabel =
+  'Editor content;Press Alt+F1 for Accessibility Options.';
+
+test.use({ storageState: 'playwright/.auth/certified-user.json' });
+
+test('User can interact with the app using the keyboard', async ({
+  page,
+  browserName
+}) => {
+  test.skip(
+    browserName === 'webkit',
+    'Failing on webkit for no apparent reason. Can not reproduce locally.'
+  );
+
+  // Enable keyboard shortcuts
+  await page.goto('/settings');
+  const keyboardShortcutsOnButton = page
+    .getByRole('button', { name: translations.buttons.on, exact: true })
+    .nth(2);
+  await keyboardShortcutsOnButton.click();
```

Comment:
```suggestion
  const keyboardShortcutGroup = page.getByRole('group', {
    name: translations.settings.labels['keyboard-shortcuts']
  });
  await keyboardShortcutGroup
    .getByRole('button', { name: translations.buttons.on, exact: true })
    .click();
```
Depending on the position in the list of 'on' buttons is a sketchy thing to rely on. If that changes, the test will fail and it won't be immediately obvious why.

The accessible name is less likely to change and, if it does, the test fails in an obvious way.

Reply:
Love it. Thanks. I'll go ahead and replace the locator in my other [PR](#52155) with this one.

Reply:
Awesome, thanks!